lldb/source/Plugins/SymbolFile/DWARF/SymbolFileDWARF.cpp

```diff
@@ -2779,7 +2788,19 @@ void SymbolFileDWARF::FindTypes(const TypeQuery &query, TypeResults &results) {
     return !results.Done(query); // Keep iterating if we aren't done.
   });
 
-  if (results.Done(query))
+  auto CheckIsDoneAndLog = [&results, &query, log, type_basename, this] {
```

Comment:
And for the one in the `-gsimple-template-names` block, it might be worth adding an something in the log that we found it during the simple-template-names (i.e., stripped template args) lookup

Reply:
Alright, I added "(simplified as <simplified_name>)" to the end of the message, but if you prefer some other format let me know